templates/Generalized_Masking.ipynb

```diff
@@ -0,0 +1,155 @@
+{
```

Comment:
<p>Similar comments here for <code>sigma</code> and <code>max_hole_size</code> being hidden default arguments.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/1092/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>